velox/type/Timestamp.h

```diff
@@ -313,10 +313,12 @@ struct Timestamp {
 
   // Assuming the timestamp represents a GMT time, converts it to the time at
   // the same moment at zone.
+  // If allowOverflow is true, integer overflow is allowed in converting
+  // timestamp to TimePoint, otherwise we throw.
```

Comment:
A tiny suggestion for comment. Thanks.

``
// @param allowOverflow If true, integer overflow is allowed when converting timestamp to TimePoint. Otherwise, user exception is thrown for overflow.
``

Reply:
Thanks for your suggestion. It's done in fa0592274fbc134bd3b75dec0a8752e74c342c5a